### DISCUSSION TITLE
How to customize input interactions

### DISCUSSION THREAD

**Discussion by evidanary** [mistic100/Photo-Sphere-Viewer#1367] [General]
How can I customize input interactions?

For example, the demo examples require 2 fingers to interact with photo sphere on mobile. Is there a way to customize input interactions so that only one finger is used?

On desktop, instead of using ctrl + scroll, I'd like to use only scroll to zoom the image.

**Comment by mistic100** [comment]
I wonder if you took 5 minutes to read the source code of the examples...

```ts
const baseUrl = 'https://photo-sphere-viewer-data.netlify.app/assets/';

const viewer = new Viewer({
    container: 'viewer',
    panorama: baseUrl + 'sphere.jpg',
    caption: 'Parc national du Mercantour <b>&copy; Damien Sorel</b>',
    loadingImg: baseUrl + 'loader.gif',
    touchmoveTwoFingers: true,
    mousewheelCtrlKey: true,
});
```

Remove the last two lines are you are good.

All of this is documented here https://photo-sphere-viewer.js.org/guide/config.html
